### DISCUSSION TITLE
V0.1.2

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#50] [公告]
此次更新花费了我不少头发，而且前后隔了很长时间（

这次还真没鸽，这段时间全程都在抽空更新

那接下来也如往常一样，让我来简述都更新了什么吧。

首先，整个主页大改，将网页捷径挪出了主界面，成为了一个“主页子页面”。

![image](https://github.com/user-attachments/assets/944187f6-9048-4eb9-92bf-afd15bd7e547)

![image](https://github.com/user-attachments/assets/a09b5936-24ea-4199-8ce0-81c564be7962)

整个主页一下变得清爽很多！~bug也少了很多~

其次，如上面第一个图那样，我把“杂项”中的一些东西移动到了主页最开头，真正的“杂项”变成了“友情链接”（可以联系我添加！）

![image](https://github.com/user-attachments/assets/8056af63-bfd6-4ff5-bc15-f334d5610f10)

“关于”界面也迎来改动，都变成了List样式！

![image](https://github.com/user-attachments/assets/cce60e14-a146-4132-b96d-630188aae1dc)

再然后，是一些细小的内容，其一是我将所有出现的中英混合内容的中英文交界处添加了空格（这个就不放图了）

其二，所有上个版本@MCSteve123用一堆减号放的线改为了<Line>元素，现在他们是实线，并且会自适应！

![image](https://github.com/user-attachments/assets/b5ed1706-88f1-4bb4-9f59-ec95cbeb237c)

其三，我将原本“作者网站”与“刷新主页”从IconButton改为IconTextButton，并找了一个更美观的位置摆放它们！并且，欢迎页的摆放更紧凑了，不像原来那样有些分散

![image](https://github.com/user-attachments/assets/61365d2d-76b9-4573-9c9e-9c093775534b)

这就是全部更新内容~哦对了，此次更新把简单主页源文件大小压缩到了164行！（包含注释，经过vscode格式化）

![image](https://github.com/user-attachments/assets/01781da8-81f6-4c2b-a977-a415f7707ecf)

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage/releases/tag/V0.1.2'>V0.1.2</a>.</em>

**Comment by MFn233** [comment]
没issue了！爽！！！！

**Reply by Ad-closeNN** [reply]
没有issue了，但是有pr了（）

**Comment by SqrtTan** [comment]
关于那些减号，其实是我没办法了，信标那里有自适应实线，我还特地去PCL那边问怎么看主页缓存……但是主页生成器生成的那个，不能说没法看，只能说太长看不懂，问信标一时半会估计也没答复，就干脆这样了（轻喷呜呜呜）

**Reply by SqrtTan** [reply]
我看看源码咋改的，学习一下

**Reply by MFn233** [reply]
直接抄信标的就行啦（bushi
